apps/client/src/components/ConclusionOutcome.tsx

```diff
@@ -0,0 +1,147 @@
+import { Button, Heading, ListItem, Paragraph } from "@datapunt/asc-ui";
+import React from "react";
+import { isIE, isMobile } from "react-device-detect";
+import { useParams } from "react-router-dom";
+
+import { ComponentWrapper, HideForPrint, List, PrintButton } from "../atoms";
+import { Olo, topics } from "../config";
+import { sections } from "../config/matomo";
+import { actions, eventNames } from "../config/matomo";
+import Link from "./Link";
+import Markdown from "./Markdown";
+
+type Props = {
+  contactConclusion: { title: string; description: string };
+  needsPermit: boolean;
+  matomoTrackEvent: Function;
+};
+
+export const ConclusionOutcome: React.FC<Props> = ({
+  contactConclusion,
+  needsPermit,
+  matomoTrackEvent,
+}) => {
+  const { slug } = useParams();
+  const topic = topics.find((t) => t.slug === slug);
+  const heading = needsPermit
+    ? "U hebt een omgevingsvergunning nodig"
+    : "U hebt geen omgevingsvergunning nodig";
+
+  const handlePrintButton = () => {
+    matomoTrackEvent({
+      action: actions.DOWNLOAD,
+      name: eventNames.SAVE_CONCLUSION,
+    });
+    window.print();
+  };
+
+  const handlePermitButton = () => {
+    matomoTrackEvent({
+      action: actions.CLICK_EXTERNAL_NAVIGATION,
+      name: eventNames.APPLY_FOR_PERMIT,
+    });
+    // Open OLO in new tab/window
+    window.open(Olo.home, "_blank");
+  };
+
+  return (
+    <>
+      <ComponentWrapper marginBottom={16}>
+        <Heading forwardedAs="h2">
+          {contactConclusion?.title || heading}.
+        </Heading>
+      </ComponentWrapper>
+      {needsPermit && (
+        <Paragraph gutterBottom={16}>
+          U kunt deze vergunning aanvragen bij het landelijk omgevingsloket
+        </Paragraph>
+      )}
+      {contactConclusion?.description && (
+        <Markdown
+          eventLocation={sections.CONCLUSION}
+          source={contactConclusion.description}
+        />
+      )}
+      <HideForPrint>
+        {needsPermit && (
+          <ComponentWrapper marginBottom={32}>
+            <Button
+              color="secondary"
+              onClick={handlePermitButton}
+              type="button"
+            >
+              Vergunning aanvragen
```

Comment:
Can you add a small test to make sure "Vergunning aanvragen" is not rendered if conclusion is not `needsPermit`? So we don't have issues with that anymore. You can do this on a separate PR, so we do the acceptance test in the meantime.